### ISSUE TITLE
Upgrade to meet HQRM Guidelines

### ISSUE THREAD

**Issue by PlagueHO** [dsccommunity/DFSDsc#21]
This resource module could be upgraded to meet HQRM guidelines.

@kwirkykat - I think this module is already fairly close to meeting the minimum standards. Is this module something that could potentially be accepted into this at some point?

**Comment by kwirkykat** [comment]
@PlagueHO Yeah it's pretty close. Looks like only some little style fixes and help comments needed.

**Comment by PlagueHO** [comment]
Thanks @kwirkykat  - What I'll do is submit a PR with the HQRM changes (based on all the changes I've been making from your feedback on my other PRs). Hopefully I'm at a point now where I know most of the things I need check. So it shouldn't be too hard.

That said, I've got so many other DSC work in the pipeline at the moment I doubt I'll get to this in the next month. So I'll look into this after I've completed the iSCSIDsc, FSRMDsc etc. Cheers!
